addons/a11y/src/components/Report/HighlightToggle.tsx

```diff
@@ -99,34 +103,55 @@ class HighlightToggle extends Component<ToggleProps> {
   private checkBoxRef = React.createRef<HTMLInputElement>();
 
   componentDidMount() {
-    this.props.elementsToHighlight.forEach(element => {
+    const { elementsToHighlight, highlightedElementsMap } = this.props;
+    elementsToHighlight.forEach(element => {
       const targetElement = getElementBySelectorPath(element.target[0]);
-      if (targetElement && !this.props.highlightedElementsMap.has(targetElement)) {
+      if (targetElement && !highlightedElementsMap.has(targetElement)) {
         this.saveElementDataToMap(targetElement, false, targetElement.style.outline);
       }
     });
   }
 
   componentDidUpdate(prevProps: Readonly<ToggleProps>): void {
+    const { indeterminate } = this.props;
     if (this.checkBoxRef.current) {
-      this.checkBoxRef.current.indeterminate = this.props.indeterminate;
+      this.checkBoxRef.current.indeterminate = indeterminate;
     }
   }
 
+  onToggle = (): void => {
```

Comment:
What are these changes?

Reply:
Linting fixes, typescript files were missing a ton of useful eslint rules!

Now all eslint rules apply to typescript code as well 🎉